src/Workspaces/CSharp/Portable/Classification/ClassificationHelpers.cs

```diff
@@ -146,6 +171,10 @@ private static string GetClassificationForIdentifier(SyntaxToken token)
                 }
                 return ClassificationTypeNames.LocalName;
             }
+            else if (token.Parent is SingleVariableDesignationSyntax singleVariableDesignation && singleVariableDesignation.Identifier == token)
```

Comment:
&& part shouldn't be necessary.  But if you're tyrin to be very paranoid, that's ok :)

Reply:
Just trying to be consistent =)